repository/CargoPackageManager-Calypso-Browser/PBCreateAssemblyCommand.class.st

```diff
@@ -0,0 +1,42 @@
+"
+Calypso command providing a contextual menu to create a new (empty) assembly into a project
+"
+Class {
+	#name : #PBCreateAssemblyCommand,
+	#superclass : #PBProjectCommand,
+	#category : #'CargoPackageManager-Calypso-Browser-Commands'
+}
+
+{ #category : #accessing }
+PBCreateAssemblyCommand class >> defaultMenuIconName [
+
+	^#newerPackagesAvailable
+]
+
+{ #category : #execution }
+PBCreateAssemblyCommand class >> execute [
+	
+	^ self name , ' Assembly'
```

Comment:
What it means?

Reply:
Yeah, this looks wrong, probably I did a mistake because of Calypso saving dirty methods...